pandas/tests/io/json/test_pandas.py

```diff
@@ -1197,6 +1198,17 @@ def test_read_local_jsonl(self):
             expected = DataFrame([[1, 2], [1, 2]], columns=["a", "b"])
             tm.assert_frame_equal(result, expected)
 
+    def test_read_json_category_dtype(self):
```

Comment:
this will need to test all extension dtypes that we support, pls parametrize this over interval, period, categorical, datetime w/tz

Reply:
Hey, according to you, what's the best way to deal/pass different input dfs and expected dfs ?

Reply:
@taoufik07 have a go using https://docs.pytest.org/en/latest/parametrize.html, there's lots of examples of it being used it other parts of the test suite

Reply:
I know about `parametrize`, but I find having multiple lines (type, input, expected) unpleasant to the eye and I tought that there was some magic function that handles the data generations for all these types, maybe I will just go with the naive way